src/tweak.cpp

```diff
@@ -175,3 +175,17 @@ UniValue settweak(const UniValue& params, bool fHelp)
     }
     return NullUniValue;
 }
+
+std::string TweakCmdLineHelp()
```

Comment:
It's nice to have some command line help for the tweaks.

It would be even better if the command line help could give information about valid range and default value of a tweak, similar to what is currently done for other cmdline args.